src/main/java/frc/robot/Robot.java

```diff
@@ -65,7 +65,7 @@ public void robotInit() {
 
         // We configure the logger here since it needs the container of all the
         // subsystems
-        Logger.configureLoggingNTOnly(this, "Logging");
+        Logger.configureLoggingNTOnly(this, "Logging"); */
```

Comment:
uncomment up until limelight, since you already initializes logger, oi, and dashboard data container. we don't care about auto-chooser